packages/client/manifest.json

```diff
@@ -5186,5 +5186,88 @@
         "suffix": "repeater"
       }
     ]
+  },
+  "rowexplorer": {
+    "block": true,
+    "name": "Row explorer block",
+    "icon": "PersonalizationField",
+    "size": {
+      "width": 100,
+      "height": 100
+    },
+    "settings": [
+      {
+        "type": "dataSource",
+        "label": "Data",
+        "key": "dataSource",
+        "required": true
+      },
+      {
+        "type": "text",
+        "label": "Height",
+        "key": "height"
```

Comment:
I think we should use a default value here, as otherwise the block is massive as soon as you add data. `426px` seems to perfectly fit 3 cards for me.